Samples/WinMLSamplesGallery/WinMLSamplesGalleryNative/.editorconfig

```diff
@@ -0,0 +1,5 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the C:\Users\numform\Windows-Machine-Learning\Samples\WinMLSamplesGallery\WinMLSamplesGalleryNative\ codebase based on best match to current usage at 12/9/2021
```

Comment:
What is this file used for?

Reply:
It's used by visual studio to maintain things like indentation for the project

Reply:
I don't think this file is needed. There's no C# coding convension rules listed even though [*.cs] is here.

Reply:
Removed.